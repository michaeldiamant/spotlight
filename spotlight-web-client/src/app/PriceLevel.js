'use strict';

import React from 'react';

class PriceLevel extends React.Component {
  render() {
    return (
      <tr> 
        <td>{this.props.price}</td>
        <td>{this.props.size}</td>
      </tr>
    );
  }
};

export default PriceLevel;
