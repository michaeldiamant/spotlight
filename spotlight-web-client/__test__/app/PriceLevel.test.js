import React from 'react';
import renderer from 'react-test-renderer';
import PriceLevel from '../../src/app/PriceLevel';

test('price level renders price and size', () => {
  var rendered = renderer.create(
    <PriceLevel price='2.01' size='1' />
  );
  expect(rendered.toJSON()).toMatchSnapshot();
});

