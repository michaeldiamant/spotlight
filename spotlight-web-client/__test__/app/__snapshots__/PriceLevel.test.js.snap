exports[`test price level renders price and size 1`] = `
<tr>
  <td>
    2.01
  </td>
  <td>
    1
  </td>
</tr>
`;
